tests/testthat/test-snapshots.R

```diff
@@ -0,0 +1,34 @@
+testthat::local_edition(3)
+quarto_docs <- list.files(
+    "../../book/",
+    recursive = TRUE,
+    full.names = FALSE,
+    pattern = "*.qmd"
+)
+
+ignore_docs <- c(
+    "appendix/session.qmd", "index.qmd", "si.qmd",
+    "test_utils.qmd", "test_save_results.qmd", "tlg-index.qmd"
+)
+
+snapshot_docs <- fs::path_file(setdiff(quarto_docs, ignore_docs))
```

Comment:
It's probably the easiest way to get the files but not so easy to maintain :P Let's say one add yet another info-type of page to the webpage and it has to be incorporated here in some strange testing file.
The whole idea about book symlink inside the package is to keep book development as loosely coupled to the testing suite as possible. I would like to keep that if possible.

Reply:
Guilty of using quick and dirty way to do this. Now we get the files explicitly by pointing to the directories: `tables`, `listings`, and `graphs`